JavaFiddle-ejb/src/main/java/ru/javafiddle/core/ejb/AccessBean.java

```diff
@@ -0,0 +1,56 @@
+package ru.javafiddle.core.ejb;
+
+import ru.javafiddle.jpa.entity.Access;
+
+import javax.ejb.Stateless;
+import javax.persistence.EntityManager;
+import javax.persistence.NoResultException;
+import javax.persistence.PersistenceContext;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Created by mac on 23.03.16.
+ */
+
+@Stateless
+public class AccessBean {
+
+    private static final Logger logger =
+            Logger.getLogger(ProjectBean.class.getName());
+
+    @PersistenceContext(name = "JFPersistenceUnit")
+    EntityManager em;
+
+    public AccessBean() {
+
+    }
+
+    public Access getAccess(String accessRights) {
+
+        Access access;
+
+        try {
+            access = (Access) em.createQuery("SELECT p FROM Access p WHERE p.accessName =:accessname")
+                    .setParameter("accessname", accessRights)
+                    .getSingleResult();
+        } catch (NoResultException noResult) {
+            logger.log(Level.WARNING, "No result in getAccess()", noResult);
+            return null;
+        }
+        return access;
+
+    }
+
+    public Access createAccess (Access access) {
+        em.persist(access);
+        em.flush();
```

Comment:
JFYI: Explicit flush is not a good practice since the code performance degrades the more flushes you have.

I know you are trying to get the entity id in the response and it is the simplest way. However, in NC we don't encourage such way (not enough scalable). T
his is more a code design question. So, let's leave as is for now.